README.md

```diff
@@ -60,9 +60,14 @@ cargo run --bin gw_configgen -- cfg /home/user/.lightning/regtest/lightning-rpc
 ```shell
 mkdir -p ln1 ln2
 ```
-Run the client config and set `ln1` as the gateway
+Run the client config and set `ln1` as the gateway. If you don't want to use Minimint with Lightning, you can skip the first command and hard-code a fake pubkey (e.g. `030160715ddbe8d4cd68e00fc910d6dac161fcfa457bfef4fd8bb93350cc2966aa`).
 ```shell
-cargo run --bin gw_configgen -- cfg "ln1/regtest/lightning-rpc"
+$ lightning-cli --network regtest --lightning-dir=ln1 getinfo
```

Comment:
This is awkward because we haven't instructed them to run lightning node yet ...

Maybe we should begin the instructions by running bitcoind, and 2 instances of lightningd? And then `startfed.sh` would only attempt to run bitdoind if it wasn't already running ...

Reply:
Ended up doing a full pass over the README. Think it's quite a bit better now ...